### ISSUE TITLE
Fix hardcoded config path

### ISSUE THREAD

**Issue by itJunky** [itJunky/web-tasker.py#13]
In `web_tasker/__init__.py`
`app = Flask(__name__, instance_relative_config=True, instance_path='/code')`

**Comment by unref** [comment]
Ух ты, как раз с этим столкнулся. Локально запускал через virtualenv. У себя локально расширил, добавил возможность указывать путь через переменную окружения, из директории проекта и из директории `/code` (как и было по дефолту).

**Comment by unref** [comment]
Мне приглянулась практика  раздиления конфига на продакшен и девелопмент. Вот тут есть пример [Development / Production](http://flask.pocoo.org/docs/0.12/config/#development-production). И вот здесь еще пример [из книги Мигеля Гринберга](https://github.com/miguelgrinberg/flasky/blob/master/config.py).

**Comment by itJunky** [comment]
Да, практика хорошая. Но пока для этого проекта не особо имеет значение. Будем реализовывать позже, когда приспичит =)
